Dockerfile

```diff
@@ -58,7 +62,7 @@ RUN curl https://sh.rustup.rs -sSf > rustup-install.sh && \
     # Install rustfmt / cargo fmt for testing
     rustup component add rustfmt clippy && \
     # Install grcov for coverage
-    cargo install grcov --version 0.8.0 --locked
+    cargo install grcov
```

Comment:
I think ideally we would still locked and pin version b/c we want this to compile like several years from now. The titanium version of this docker file is a real nightmare to work with b/c nothing installs/builds properly in there, due to breaking updates.

Reply:
FYI, it does compile successfully without `--locked`, but with `--locked`,  there is currently one warning:
```
 Downloading crates ...                                                                                                                                                                                                                                                                                                                                                                                                          
  Downloaded grcov v0.8.4                                                                                                                                                                                                                                                                                                                                                                                                        
  Installing grcov v0.8.4                                                                                                                                                                                                                                                                                                                                                                                                        
warning: package `crossbeam-deque v0.8.0` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked        
```

Reply:
I think we can live with that warning. I don't think grcov is meaningfully affected by the yank.

Reply:
Fixed.  Updated version uses `--locked` and is pinned to `0.8.4`